ape_etherscan/client.py

```diff
@@ -8,6 +8,7 @@
 from ape.logging import logger
 from ape.utils import USER_AGENT, ManagerAccessMixin
 from requests import Session
+from yarl import URL
```

Comment:
Should probably use this in the RPC debug in core because it's currently showing my project id lol

Reply:
hmm , I thought we were already!

Maybe this is happening in the foundry plugin when connecting to the upstream provider?
Show me where if you find it. I know ape-geth handles it well but it may be missed elsewhere.

Reply:
it was doing it for me w/ infura at least

Reply:
I am having troubles locating where this might be happening. Can you share with me what the log looks like?